packages/admin-ui/components/infoModal/MetricInfo.tsx

```diff
@@ -1,5 +1,6 @@
 import { useMemo } from 'react';
-import { METRIC_TYPE, MetricData, TimeGrain } from '@vulcan-sql/admin-ui/utils/data/model';
+import { METRIC_TYPE } from '@vulcan-sql/admin-ui/utils/enum';
+import { MetricData, TimeGrain } from '@vulcan-sql/admin-ui/utils/data/model';
```

Comment:
Will we prefer use `'@vulcan-sql/admin-ui/utils/data'` or `'@vulcan-sql/admin-ui/utils/data/model'`?